cc_licenses/static/502.html

```diff
@@ -33,3 +33,4 @@ <h1>Site Maintenance</h1>
         <p>The site will be back to normal shortly.</p>
     </div>
 </body>
+{# vim: ft=jinja.html ts=2 sw=2 sts=2 sr et #}
```

Comment:
While these comments are minimal, we might consider how IDE metadata can live in a project-wide configuration file in the project root, rather than through the project code. E.g. `.vscode` directory contains VSCode configuration. At the very least, it would align with the DRY principle.

What is your experience with `.vimrc` or other similar approaches to configuring project-wide Vim settings?

https://akrabat.com/using-vimrc-for-project-specific-settings/